charts/cluster-security/templates/bindings.yaml

```diff
@@ -1,30 +1,36 @@
-# This role binding allows all pods in the 'default' and 'apps' namespace to use the baseline PSP.
-apiVersion: rbac.authorization.k8s.io/v1
+ {{- range .Values.baselineNamespaces }}
 kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: psp-baseline-rolebinding
-  namespace: apps
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
   name: psp-baseline
+  namespace: {{ . }}
 subjects:
 - kind: Group
-  name: system:authenticated
-  namespace: apps
+  name: system:serviceaccounts:{{ . }}
   apiGroup: rbac.authorization.k8s.io
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
-metadata:
-  name: psp-baseline-rolebinding
-  namespace: default
 roleRef:
-  apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: psp-baseline
+  apiGroup: rbac.authorization.k8s.io
+---
+{{ end }}
+---
+{{- range .Values.privilegedNamespaces }}
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: psp-privileged
+  namespace: {{ . }}
 subjects:
 - kind: Group
-  name: system:authenticated
-  namespace: default
+  name: system:serviceaccounts:{{ . }}
   apiGroup: rbac.authorization.k8s.io
+- kind: Group
+  name: system:cluster-admins
```

Comment:
Are you sure this group exists? (i.e. does it actually contain any principals?)

Reply:
I checked `k get -n kube-system configmap/aws-auth -o yaml`
There is a "cluster-admins" group that is populated with gds team, but it might not need the "system:" prefix.
I'll change it.

Reply:
Cool, thanks. Yeah it's a different group with the `system:` prefix. The string has to match.